core/src/main/scala/kafka/server/DynamicConfigs.scala

```diff
@@ -0,0 +1,89 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements.  See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License.  You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package kafka.server
+
+import java.util.Properties
+
+import org.apache.kafka.common.config.ConfigDef
+import org.apache.kafka.common.config.ConfigDef.Importance._
+import org.apache.kafka.common.config.ConfigDef.Type._
+
+/**
+  * Class used to hold dynamic configs. These are configs which have no physical manifestation in the server.properties
+  * and can only be set dynamically.
+  */
+object DynamicConfigs {
+
+  object Broker {
+    //Properties
+    val ThrottledReplicationRateLimitProp = "replication.quota.throttled.rate"
+
+    //Defaults
+    val DefaultThrottledReplicationRateLimit = ReplicationQuotaManagerConfig.QuotaBytesPerSecondDefault
+
+    //Documentation
+    val ThrottledReplicationRateLimitDoc = "A long representing the upper bound (bytes/sec) on replication traffic for replicas enumerated in the " +
+      s"property $ThrottledReplicationRateLimitProp. This property can be only set dynamically via the config command etc."
+
+    //Definitions
+    val brokerConfigs = new ConfigDef()
+      .define(ThrottledReplicationRateLimitProp, LONG, DefaultThrottledReplicationRateLimit, MEDIUM, ThrottledReplicationRateLimitDoc)
```

Comment:
Is it correct that you removed the `atLeast(0)` validator because we only validate names via this class? Would it make sense to define other validation here as well? That would make it more consistent with how we handle the static configs.